### ISSUE TITLE
[REQUEST] min-range and max-range option on Range component

### ISSUE THREAD

**Issue by skmbr** [quasarframework/quasar#2100]
I would like to be able to place limits on the size of the range the user can select.

eg. On a range slider of 1 to 100 I might set min-range="5" and max-range="20" so that the range selected can only between 5 and 20

**Comment by mesqueeb** [comment]
That's the `:min` and `:max` right?
```html
  <q-range
    v-model="rangeValues"
    :min="5"
    :max="20"
    :step="1"
  />
```
https://quasar-framework.org/components/range.html

**Comment by mesqueeb** [comment]
What's your use case?

**Comment by rstoenescu** [comment]
@skmbr Hi, You could validate user input and signal an error if the range is incorrect (`error` prop). Trying to keep the component as lightweight as possible so it won't affect render speed..

**Comment by rstoenescu** [comment]
For anyone landing here: this got revisited and implemented after all. `min-range` and `max-range` props for QRange landed on `dev` (quasarframework/quasar@eb9191121e2ef37a48ea1a9d84e0dee8de4c5abb, based on PR #18182 by @thexeos) and will ship in v2.26.0.
